### ISSUE TITLE
Translate: parent= should not be a required argument for every function call

### ISSUE THREAD

**Issue by verhovsky** [googleapis/google-cloud-python#9578]
In the majority of use cases it's set once from the credentials. This is what object properties are for.

**Comment by tseaver** [comment]
@busunkim96, @lukesneeringer The addition of the `google.api_method.signature` annotations in [this commit](https://github.com/googleapis/googleapis/commit/662bfb2666b4310027475928234f1355dcf8a935) is the source of the required `parent` argument.  ISTM that @verhovsky is correct, and that the client should default `project` to the value from its own credentials if it is not passed explicitly.

**Comment by busunkim96** [comment]
@nnegrey Is there some reason that `project` must be passed with every call with the translate API?

v3beta1 also seems to require `parent` for every call https://cloud.google.com/translate/docs/reference/rpc/google.cloud.translation.v3beta1#batchtranslateresponse

**Comment by nnegrey** [comment]
Yes, but it isn't just `project`, there is also a `location` that needs to be passed as well
```
parent = client.location_path(project_id, "global")
```

**Comment by verhovsky** [comment]
You can have more than one object property.

But also in the the above code sample I'm using a string without a location and it works fine
